### PR TITLE
Features/connect policy evaluation to postgresql

### PR DESCRIPTION
**Pull request by nam-m**
## Description

Issue Link: #43 

I created Dockerfile to build the container for `airbnb_project` from poetry dependencies, also made it multi-stage to minimize build artifacts. And added it to docker-compose under `policies` app, ideally, there should be a container for each app (listings, policies, dashboard) and 1 for db (already made).
I just ran simple `python manage.py check` to check the `policies` app after all docker setup since the service is not created yet.

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [ ] Unit Test
- [ ] Integration Test
- [ ] E2E Test

## Checklist:

Before you submit your pull request, please make sure you have completed the following:

- [x] I have read the CONTRIBUTING document.
- [x] I have checked that my code adheres to the code style of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

Here's the pic of `db` and `policies` containers running in Docker Desktop
image

## Additional Notes

Add any other notes about the pull request here.

### CODE REVIEW

File: airbnb_project/airbnb_project/settings.py [hunk 1]
```diff
@@ -82,6 +82,10 @@
     'default': {
         'NAME': 'airbnb_db',
         'ENGINE': 'django.db.backends.postgresql',
+        'USER': os.environ['POSTGRES_USER'],
```
Comment (VimanyuAgg) [inline airbnb_project/airbnb_project/settings.py:85]: `nit`: It'd be nice to have an env file where we can find out what all environment variables are needed. Not blocking on it though.

**Review by VimanyuAgg — APPROVED** [review]
Thanks a lot for the efforts, @nam-m ! I don't have any blocking comments.

File: airbnb_project/airbnb_project/settings.py (hunk 1 above)
Comment (nam-m) [reply airbnb_project/airbnb_project/settings.py:85]: @VimanyuAgg thanks for your feedback, could it be an env template file like:
```
SECRET_KEY=
DJANGO_DEBUG=
AIRBNB_PUBLIC_API_KEY=
POSTGRES_USER=
POSTGRES_PASSWORD=
POSTGRES_DB=
POSTGRES_HOST_PORT=
```

Comment (nam-m) [reply airbnb_project/airbnb_project/settings.py:85]: i will still merge this pr to let Mangat use Dockerfile

**Pull request merged**